roles/splunk_common/tasks/add_splunk_license.yml

```diff
@@ -13,31 +23,13 @@
   retries: 5
   delay: 3
```

Comment:
Probably should do this how I do the "pre-play / post-play" and do a first found lookup, and download to a set location.  Don't forget the nofail on the download incase there's a file path.